Dockerfile-GordoDeploy

```diff
@@ -56,5 +56,9 @@ COPY resources /code/resources
 COPY --from=builder /code/dist/gordo-infrastructure-packed.tar.gz .
 RUN pip install ./gordo-infrastructure-packed.tar.gz
 
+# Nonroot user for running CMD
+RUN groupadd -g 999 gordo && \
+    useradd -r -u 999 -g gordo gordo
+USER gordo
 
 CMD ["bash", "/code/run_workflow_and_argo.sh"]
```

Comment:
you probably want to `chown /code` folder also

Reply:
It has `rx` for `ugo` already
```bash
-rwxrwxr-x  1 rdi rdi    538 Oct 22 10:05 run_workflow_and_argo.sh
```
..but it does look like I need to `chown` more for the catboost library to do its thing in the `ModelBuilder` image.
```bash